documents/knowledge/yaml_to_txt.py

```diff
@@ -3,7 +3,7 @@
 
 CURR_DIR = os.path.dirname(os.path.realpath(__file__))
 ELEMENTS_YAML = os.path.join(CURR_DIR, "elements.yaml")
-ELEMENTS_TXT = os.path.join(CURR_DIR, "elements.txt")
+ELEMENTS_TXT = os.path.join(CURR_DIR, "Elements.txt")
```

Comment:
Would it be better to just rename `Elements.txt` to be lowercase across the board, for consistency with the others?

Reply:
Renaming it would be worth it

Reply:
Yes - several things link to `elements.txt` which doesn't exist